### ISSUE TITLE
Execution halted due to Error in nlminb

### ISSUE THREAD

**Issue by GilbertHeywood** [James-Thorson-NOAA/VAST#201]
>processing file: VAST--single-species_example.Rmd
>Quitting from lines 304-305 (VAST--single-species_example.Rmd) Error in nlminb(start = startpar, objective = fn, gradient = gr, control = nlminb.control,  : gradient function must return a numeric vector of length 75 Calls: <Anonymous> ... withVisible -> eval -> eval -> <Anonymous> -> fit_tmb -> nlminb
>Execution halted

Does anyone know how to resolve this issue?

I'm using R version 3.6.1, g++ 9.1, VAST 3.2.1.

**Comment by colemonnahan** [comment]
I think that example script is outdated. It's better to try the models on the wiki, e.g., 

https://github.com/James-Thorson-NOAA/VAST/wiki/Simple-example

Does that run for you?

**Comment by GilbertHeywood** [comment]
> I think that example script is outdated. It's better to try the models on the wiki, e.g.,
> 
> https://github.com/James-Thorson-NOAA/VAST/wiki/Simple-example
> 
> Does that run for you?

Yes, I can. But it is less descriptive than the Rmarkdown. Is there any other tutorial-like code for me to understand how to run the code? Thx!

**Comment by James-Thorson** [comment]
I'd like to learn more about your goals and use-case before responding.

Why do you want to learn more about those "mid-level" function in the old
Rmarkdown tutorials rather than the "high-level" functions in the wiki
vignettes?

The high-level functions include all functionality and are easier to
document using R reference docs.
